vic/drivers/shared_image/include/vic_mpi.h

```diff
@@ -29,6 +29,11 @@
 
 #include <vic_def.h>
 #include <mpi.h>
+#ifdef _OPENMP
```

Comment:
document this setting in the build documentation

Reply:
I have. This is set when the `-openmp` flag is used. Users don't need to know about this particular macro though.

Reply:
Oh - I see - even better.